inc/functions.php

```diff
@@ -12,11 +12,15 @@
 /**
  * Display the main Juicer social media feed.
  *
- * @param int $count The number of posts to display.
- * @param int $page  The page to display.
+ * @param int  $count     The number of posts to display.
+ * @param int  $page      The page to display.
+ * @param bool $load_more Whether to include a load more button.
  */
-function juicer_feed( $count = 10, $page = 1 ) {
+function juicer_feed( $count = 10, $page = 1, $load_more = false ) {
```

Comment:
We can typecast these parameters here, too.

```suggestion
function juicer_feed( int $count = 10, int $page = 1, bool $load_more = false ) {
```

Reply:
Fixed in [e265b3b](https://github.com/humanmade/hm-juicer/pull/6/commits/e265b3bb7c03ddea9243b79b7fac536dccd671d3)